doc/source/whatsnew/v0.10.0.rst

```diff
@@ -60,7 +53,7 @@ talking about:
    # deprecated now
    df - df[0]
    # Change your code to
-   df.sub(df[0], axis=0) # align on axis 0 (rows)
+   df.sub(df[0], axis=0)    # align on axis 0 (rows)
```

Comment:
```suggestion
   df.sub(df[0], axis=0)  # align on axis 0 (rows)
```
just two spaces as pep8 says (with the `noqa` I've seen we leave more, and I guess it makes sense, as we don't want users to see them much